client/state/themes/actions/request-themes.js

```diff
@@ -50,6 +50,7 @@ export function requestThemes( siteId, query = {}, locale ) {
 						{
 							...query,
 							apiVersion: '1.2',
+							unlock_blankcanvas_theme: config.isEnabled( 'pattern-assembler/logged-out-showcase' ),
```

Comment:
I'd suggest following the existing pattern of `include_marketplace_themes`, so e.g. `include_blank_canvas_themes`, what do you think?

Reply:
I see you